lib/main.dart

```diff
@@ -1,14 +1,286 @@
 import 'package:flutter/material.dart';
+import 'package:chip_list/chip_list.dart';
+import '../shared/constants.dart' as app_constants;
 
-void main() {
-  runApp(MyApp());
-}
+void main() => runApp(const MyApp());
 
 class MyApp extends StatelessWidget {
+  const MyApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return const MaterialApp(home: Home());
+  }
+}
+
+class Home extends StatefulWidget {
+  const Home({super.key});
+
+  @override
+  State<Home> createState() => _HomeState();
+}
+
+class _HomeState extends State<Home> {
+  int _likeCounter = 0;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      resizeToAvoidBottomInset: false,
+      appBar: AppBar(
+        title: const Text(
+          app_constants.ConstantsClass.appName,
+          style: TextStyle(
+            fontFamily: app_constants.ConstantsClass.appFont,
+            fontSize: 30.0,
+            fontWeight: FontWeight.bold,
+          ),
+        ),
+        centerTitle: true,
+        backgroundColor: Colors.red,
+        leading: const Icon(Icons.menu_rounded),
+      ),
+      body: SafeArea(
+        child: Container(
+          color: Colors.black,
+          child: ListView(
+            children: [
+              const _Header(),
+              const _SubHeader(),
+              _Description(),
+            ],
+          ),
+        ),
+      ),
+      floatingActionButton: FloatingActionButton.extended(
+        onPressed: () => setState(() => _likeCounter++),
+        backgroundColor: Colors.red,
+        splashColor: Colors.red[200],
+        //When you click on the button, the color changes
+        label: Row(
+          children: [
+            const Icon(
+              Icons.thumb_up,
+              color: Colors.white,
+            ),
+            Text(
+              ' $_likeCounter',
+              style: const TextStyle(
+                color: Colors.white,
+              ),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class _Description extends StatelessWidget {
+  final List<String> _genres = [
+    'Adventure',
+    'Animation',
+    'Comedy',
+    'Family',
+    'Fantasy',
+  ];
+
+  final String voteAverage = '7.8',
```

Comment:
I think this can be const instead of final

Reply:
solved